.travis.yml

```diff
@@ -31,4 +33,4 @@ deploy:
     tags: true
     all_branches: true
     repo: flaskbb/flaskbb
-    python: '3.6'
+    python: '3.7'
```

Comment:
Since we have two 3.6 envs doing the pypi deploy off that env will cause at least one of the builds to fail when pypi rejects the deploy.